Frameworks/GLKit/GLKTexture.mm

```diff
@@ -277,7 +279,8 @@ @implementation GLKTextureLoader {
 */
 + (GLKTextureInfo*)textureWithContentsOfFile:(NSString*)fname options:(NSDictionary*)opts error:(NSError**)err {
     CGDataProviderRef provider = CGDataProviderCreateWithFilename([fname UTF8String]);
-    CGImageRef img = CGImageCreateWithPNGDataProvider(provider, NULL, NO, kCGRenderingIntentDefault);
+    unsigned char* dataBytes = static_cast<unsigned char*>(const_cast<void*>(_CGDataProviderGetData(provider)));
```

Comment:
hmm looks like others functions too, loadJPEG, etc.


---
In reply to: [103336909](https://github.com/Microsoft/WinObjC/pull/2106#discussion_r103336909) [](ancestors = 103336909)